File: app/src/main/java/com/mercadolibre/android/mlbusinesscomponentsapp/MLBusinessAdBannerDataSample.java
package com.mercadolibre.android.mlbusinesscomponentsapp;

import androidx.annotation.NonNull;

import com.mercadolibre.android.mlbusinesscomponents.components.adbanner.MLBusinessAdBannerData;

public class MLBusinessAdBannerDataSample implements MLBusinessAdBannerData {
    @NonNull
    @Override
    public String getImageUrl() {
        return "https://http2.mlstatic.com/D_NQ_750106-MLA46995964779_082021-F.webp";
    }

    @NonNull
    @Override
    public String getUrlDeepLink() {
        return "https://www.mercadolibre.com.ar/";
    }
}

File: app/src/main/java/com/mercadolibre/android/mlbusinesscomponentsapp/MLBusinessAdBannerDataSample.kt
package com.mercadolibre.android.mlbusinesscomponentsapp

import com.mercadolibre.android.mlbusinesscomponents.components.adbanner.MLBusinessAdBannerData

class MLBusinessAdBannerDataSample : MLBusinessAdBannerData {
    override val imageUrl: String
        get() = "https://http2.mlstatic.com/D_NQ_750106-MLA46995964779_082021-F.webp"
    override val urlDeepLink: String
        get() = "https://www.mercadolibre.com.ar/"
}
